### DISCUSSION TITLE
Welcome to TDA_CrashDetection Discussions!

### DISCUSSION THREAD

**Discussion by FloerHogy** [Laura-gr/TDA_CrashDetection#3] [Announcements]
Main discussion

**Comment by todortodor** [comment]
Need to list tickers

**Comment by FloerHogy** [comment]
Added the scalings to the branch and the plot of presistence diagram and computation of persistence landscape.
Pers. landscape is a np.array so should be easy to plot with plotly, but not yet done
Pers. diagr plot comes with gudhi package so don't know how to get it work with plotly, have to look into it
